src/test/json/json-patch/add_something_test1.json

```diff
@@ -0,0 +1,13 @@
+[ {
```

Comment:
Let's rename this test file more appropriately. Ideally mentioning where we saw it - per the naming conventions of the other files. 

Maybe `add_members_nested_schema_azure2.json` -or optionally add to the existing `add_members_azure1` test.